bentoml/handlers/image_handler.py

```diff
@@ -100,4 +104,19 @@ def handle_cli(self, args, func):
         print(result)
 
     def handle_aws_lambda_event(self, event, func):
-        raise NotImplementedError
+        try:
+            import cv2
+        except ImportError:
+            raise ImportError("opencv-python package is required to use ImageHandler")
+
+        if event['headers']['Content-Type'] in ACCEPTED_CONTENT_TYPES:
```

Comment:
when Content-Type header is not set by client request, the AttributionError is not handled. maybe use `if event['headers'].get('Content-Type', None) in ACCEPTED_CONTENT_TYPES:` here instead?

Reply:
Good suggestion